### ISSUE TITLE
Save your work feature enhancement

### ISSUE THREAD

**Issue by joshuawalcher** [joshuawalcher/writtenkitten#17]
Users should be able to:

1) Register with the website with a simple username and password only (email optional, but required for resetting password)
2) Save their work automatically as well as by button press.

**Comment by hoodedice** [comment]
This is a client-side only application right now, does it mean you plan to add in a backend?

**Comment by joshuawalcher** [comment]
Yes, we'll be adding a backend.

The database schema will be added to the project and checked in, and there will be a config file that will be .gitignore'd, though a default blank config file will be checked in before the ignore.

**Comment by joshuawalcher** [comment]
Branch created. Work underway: https://github.com/joshuawalcher/writtenkitten/tree/issue-17-save-your-work-enhancement

**Comment by joshuawalcher** [comment]
Database documented in wiki. https://github.com/joshuawalcher/writtenkitten/wiki/MySQL-Database

**Comment by joshuawalcher** [comment]
70% done.

**Comment by davidhammaker** [comment]
@joshuawalcher 
Is the PHP / MySQL backend still in the works? A Python backend could be a great solution.

**Comment by a2937** [comment]
I don't think you need users and authentication just yet. While nice, you probably should add a copy, paste, and download button first.
